CHANGELOG.md

```diff
@@ -1,6 +1,27 @@
 Note: This is in reverse chronological order, so newer entries are added to the top.
 
 
+Swift 5.3.1
+-----------
+
+* [#2937](https://github.com/apple/swift-package-manager/pull/2937)
+  
+  Manifest files can now have any combination of leading line terminators. All [Unicode line terminators](https://www.unicode.org/reports/tr14/) are recognised. Gone are ye olde days when a shebang-like directive must be at the start of file.
+  
+  [SR-13566](https://bugs.swift.org/browse/SR-13566) The Swift tools version specification in each manifest file now accepts any combination of _horizontal_ whitespace characters between `//` and `swift-tools-version`, if and only if the specified version > 5.3. For example, `//swift-tools-version:5.3.1` and `//		 swift-tools-version:5.4` are valid.
+  
+  * Deprecations
+    
+    `ToolsVersionLoader.split(_ bytes: ByteString) -> (versionSpecifier: String?, rest: [UInt8])` is now deprecated, and replaced by `ToolsVersionLoader.split(_ manifestContents: String) -> ManifestComponents`.
+  
+    `ToolsVersionLoader.Error.malformedToolsVersion(specifier: String, currentToolsVersion: ToolsVersion)` is not deprecated, and replaced by `ToolsVersionLoader.Error.malformedToolsVersionSpecification(_ malformation: ToolsVersionSpecificationMalformation)`.
+  
+  * Source Breakages
+    
+    SPM now throws an error if a manifest file contains invalid byte sequences such as `0x7F8F`.
```

Comment:
Also, I think it might be sufficient to say "invalid UTF-8 byte sequences" rather than mentioning a particular code, so nobody gets the idea that there's something special about that code.

Reply:
In 6e865967063f1e1b6be4537d3a20fd61d7dc2493, I replaced "SPM" with "The package manager" instead of either "SwiftPM" or "The Swift Package manager", because it's what the Swift 3.0 changelog entry uses. I can change both of them to the suggested terms, if the replacement isn't correct or precise.

In 147bd55295f44e0729418fcd0946f86c79f5e9b3, I replaced "invalid byte sequences such as `0x7F8F`" with "invalid UTF-8 byte sequences" verbatim, and credited you as a co-author.

Reply:
Thanks a lot!  I realize that the existing terminology is quite inconsistent already, so there aren't a lot of good examples to look at.  Your changes look great.